packages/client/tests/functional/_utils/getTestSuiteInfo.ts

```diff
@@ -35,11 +39,10 @@ export function getTestSuiteFullName(suiteMeta: TestSuiteMeta, suiteConfig: Name
  * @param suiteConfig
  * @returns
  */
-export function getTestSuitePreviewFeatures(matrixOptions: Record<string, string>) {
-  return [
-    ...(matrixOptions['providerFeatures']?.split(', ') ?? []),
-    ...(matrixOptions['previewFeatures']?.split(', ') ?? []),
-  ]
+export function getTestSuitePreviewFeatures(schema: string): string[] {
+  const match = schema.match(schemaPreviewFeaturesRegex)
+
+  return match === null ? [] : JSON.parse(match[1])
```

Comment:
Is it possible that `match[1]` is undefined?

Reply:
On a type-level in theory yes, and in fact not because the regex has a group.